### DISCUSSION TITLE
mPdf Using a ‘cropped’ page not working

### DISCUSSION THREAD

**Discussion by slaouiyounes** [mpdf/mpdf#1784] [Q&A]
I am trying to import part of a page using the following code
`$pageCount      =   $this->mpdf->setSourceFile(test.pdf');`
`$tplIdx 	    = 	$this->mpdf->importPage($pageCount);`
`$this->mpdf->useTemplate($tplIdx, 0, 0, 50, 50);`
The result
![image](https://user-images.githubusercontent.com/45993397/209781206-ca266cc8-b5b0-4f39-a98c-b0f2688f24f4.png)
In the previous version you could pass parameters in the ImportPage function

**Comment by finwe** [comment]
What are you trying to achieve by spamming duplicate threads?

**Reply by slaouiyounes** [reply]
I made a mistake in the channel and I can no longer delete the message

**Reply by finwe** [reply]
Then: 
- Provide a reproducible example incl. the imported PDF
- What "previous version" are you referencing? You can still pass parameters to the ImportPage function, but as I tried out, they only limit dimensions of the imported PDF, not the "cutout".

**Comment by slaouiyounes** [comment]
thank you for your reply
In mpdf version 6 I had the following code
`$mpdf->SetImportUse();`
`$mpdf->SetSourceFile('test.pdf');`
`$tplIdx 	= 	$mpdf->ImportPage(1, 0, 0, 210, 60);`
`$mpdf->UseTemplate($tplIdx, 0, 0, 210, 60); `
The original file that contains a single page
![image](https://user-images.githubusercontent.com/45993397/209924226-4c0ce9ee-e9c7-4651-a502-c7a7405b6e13.png)
Result on mpdf version 6
![image](https://user-images.githubusercontent.com/45993397/209924318-4919c811-372f-4fa1-b729-81e5ce570241.png)

In the latest version mpdf
`$mpdf->setSourceFile('test.pdf');`
`$tplIdx 	= 	$mpdf->importPage(1);`
`$mpdf->useTemplate($tplIdx, 0, 0, 210, 60);`
Result on the latest version mpdf
![image](https://user-images.githubusercontent.com/45993397/209924824-79ed3068-dd9d-4f76-8117-8eb6161de4ed.png)
The whole page has been compressed
